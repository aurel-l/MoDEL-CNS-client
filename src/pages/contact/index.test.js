import React from 'react';
import { shallow } from 'enzyme';

import Contact from '.';

describe('<Contact />', () => {
  it('should render', () => {
    expect(shallow(<Contact />)).toMatchSnapshot();
  });
});
