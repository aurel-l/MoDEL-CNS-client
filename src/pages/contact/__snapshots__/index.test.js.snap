// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Contact /> should render 1`] = `
<WithStyles(ForwardRef(Card))
  className="card"
>
  <WithStyles(ForwardRef(CardContent))
    className="card-content"
  >
    <WithStyles(ForwardRef(Typography))
      variant="h5"
    >
      Get in touch with us
    </WithStyles(ForwardRef(Typography))>
    <a
      href="http://mmb.irbbarcelona.org/www/locationmap"
      rel="noopener noreferrer"
      target="_blank"
    >
      See on the IRB Molecular Modeling and Bioinformatics group website
    </a>
  </WithStyles(ForwardRef(CardContent))>
</WithStyles(ForwardRef(Card))>
`;
